polyfills.js

```diff
@@ -99,7 +99,7 @@ function patch (fs) {
             && Date.now() - start < 60000) {
           setTimeout(function() {
             fs.stat(to, function (stater, st) {
-              if (stater && stater.code === "ENOENT")
+              if ((stater && stater.code === "ENOENT") || (st && st.isFile()))
```

Comment:
Could you explain this fix, maybe add a comment?  Is the situation that `to` already exists but is maybe open by another process so windows gives EACCES or EPERM when trying to overwrite the destination?

Reply:
Yes, as explained in this https://github.com/isaacs/node-graceful-fs/issues/104#issue-198434542 , after the [stat check](https://github.com/isaacs/node-graceful-fs/commit/90a96bc2104729e51d5d7b4aebc8c584ccf7d2ce) was added, he started seeing these errors. `to` already exists because it is a config file that is being atomically renamed to (overwritten). The stat check was added because the retry mechanism was causing a long retry delay in the case that `to` is a directory because it would give the same error and thereby retry until timeout. Therefore with `st.isFile()` it should still give up on directories.